pkg/rke2/rke2_linux_test.go

```diff
@@ -0,0 +1,37 @@
+//go:build linux
+// +build linux
+
+package rke2
+
+import (
+	"flag"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
Thanks for adding a unit test!! However, we don't use testify in our Unit tests, we generate them via `gotests` tool. Please convert this over to that format. See https://github.com/rancher/rke2/blob/master/developer-docs/testing.md#unit-tests for more details.

Reply:
I wrap this in a table-driven design, but I don't see that use of table-driven tests are mutually exclusive with use of `assert` to check the actual values of the returned data. gotest still leaves it up to you to check the type and value of your data against the desired output, create an error message, and so on. The basic assert functions are a very useful shorthand for that.